packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -434,13 +446,18 @@ export class EntityInsertPanelImpl extends ReactN.Component<Props, StateProps> {
                     };
                 },
                 () => {
+                    if (this.props.onParentChange) {
+                        this.props.onParentChange(updatedModel.entityParents);
+                    }
```

Comment:
could be simplified using the `?.` syntax, I think it is:
```suggestion
                    this.props?.onParentChange(updatedModel.entityParents);
```

Reply:
@cnathe It would actually be:

```ts
this.props.onParentChange?.(updatedModel.entityParents);
```